Extension/PesterTask/PesterV10/Pester.ps1

```diff
@@ -110,30 +109,27 @@ if ($ExcludeTag) {
     $Filter.Add('ExcludeTag', $ExcludeTag)
 }
 
-$PesterConfig['Filter'] = $Filter
+$PesterConfig.Filter = $Filter
 
-$CodeCoverage = @{}
 if ($CodeCoverageOutputFile) {
-    $CodeCoverage['Enabled'] = $True
-    $CodeCoverage['OutputFormat'] = "JaCoCo"
+    $PesterConfig.CodeCoverage.Enabled = $True
+    $PesterConfig.CodeCoverage.OutputFormat = "JaCoCo"
 
     if (-not $PSBoundParameters.ContainsKey('CodeCoverageFolder')) {
         $CodeCoverageFolder = $TestFolder
     }
     $Files = Get-ChildItem -Path $CodeCoverageFolder -include *.ps1, *.psm1 -Exclude *.Tests.ps1 -Recurse |
-        Select-object -ExpandProperty Fullname
+    Select-object -ExpandProperty Fullname
 
     if ($Files) {
-        $CodeCoverage.Add('Path', $Files)
-        $CodeCoverage.Add('OutputPath', $CodeCoverageOutputFile)
+        $PesterConfig.CodeCoverage.Path = $Files
+        $PesterConfig.CodeCoverage.OutputPath = $CodeCoverageOutputFile
     }
-    else {
+    else {`
```

Comment:
Looks like a random backtick ended up here, can you remove it? It won't cause any harm being there with nothing after it but it's also not doing anything.

Reply:
Completed.